### ISSUE TITLE
[Bug]: OptOut and agian Opt is not sending notification anymore

### ISSUE THREAD

**Issue by nayanAubie** [OneSignal/OneSignal-Flutter-SDK#729]
### What happened?

Call the `optOut()` method and again call the `optIn()` will not send notification anymore. after that, I need to uninstall the app and reinstall it to work properly.
Everything worked perfectly until I logged out from the account and called `optOut()`

### Steps to reproduce?

On user logout operation, set the below code:

```dart
await OneSignal.User.pushSubscription.optOut();
await OneSignal.User.removeTags(['name', 'email']);
await OneSignal.logout();
```

on user login operation, set the below code:
```dart
await OneSignal.login('${user.email}');
OneSignal.User.pushSubscription.optIn();
```

### What did you expect to happen?

It should send the notification again when `optIn()`

### OneSignal Flutter SDK version

onesignal_flutter: ^5.0.0

### Which platform(s) are affected?

- [X] iOS
- [X] Android

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by emawby** [comment]
@nayanAubie Thank you for reaching out I am not able to reproduce this issue. What is the subscription's status on the dashboard? Is it a checkmark or does it say something else?

**Comment by emawby** [comment]
@fanwgwg Before you call optOut is the subscription enabled in the dashboard? Does the subscription also have a push token and push permissions enabled?

**Comment by nan-li** [comment]
Hi everyone, thank you for reporting and your patience. We released the fix in [Release 5.1.0](https://github.com/OneSignal/OneSignal-Flutter-SDK/releases/tag/5.1.0).

Please update and reach out if you are still seeing issues.
